### PR TITLE
Allow adjusting the number of replicas for existing services

### PR DESCRIPTION
**Pull request by cwurm**
Add `num_replicas` field to control the number of replicas (field `numReplicas` in `/scaling`: https://clickhouse.com/docs/en/cloud/manage/api/swagger#/paths/~1v1~1organizations~1:organizationId~1services~1:serviceId~1scaling/patch).

### CODE REVIEW

**Comment by juan-vg** [comment]
Since [the TF docs](https://github.com/ClickHouse/terraform-provider-clickhouse/blob/main/docs/resources/service.md) have been manually refactored, I believe it would make sense to add this new field there

**Comment by cwurm** [comment]
@juan-vg Good point, done.

File: docs/resources/service.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ resource "clickhouse_service" "this" {
 - `idle_timeout_minutes` (Number) Set minimum idling timeout (in minutes). Available only for 'production' services. Must be greater than or equal to 5 minutes.
 - `max_total_memory_gb` (Number) Maximum total memory of all workers during auto-scaling in Gb. Available only for 'production' services. Must be a multiple of 12 and lower than 360 for non paid services or 720 for paid services.
 - `min_total_memory_gb` (Number) Minimum total memory of all workers during auto-scaling in Gb. Available only for 'production' services. Must be a multiple of 12 and greater than 24.
+- `num_replicas` (Number) Number of replicas for the service. Available only for 'production' services. Must be greater than 3 and less than 25. Contact support to enable this feature.
```
Comment (rndD) [inline docs/resources/service.md:44]: Let's add `num_replicas cannot be defined on a new service, only on existing services.` also here.

**Review by rndD — APPROVED** [review]
Looks good, I tested it.

**Comment by juan-vg** [comment]
I would also add a check for tier development, ensuring is not different from null

**Pull request merged**